Source/Documentation/Manual/appendices.rst

```diff
@@ -23,7 +22,7 @@ Measure                 Default unit    Applies to  OR accepts  MSTS accepts
 ======================= =============== =========== =========== =============== ==============================
 Mass                    kg                          kg          kg                
 \                                                   t           t               metric tonne (1000 kg)
-\                                                   lb          lb                
+\                                                   lb          lb                                    
```

Comment:
Those spaces are most probably not needed.

Reply:
Yes, they crept in unobserved. Fixed now.